Makefile

```diff
@@ -23,6 +23,13 @@ BUILD_TIME=$(shell date -u '+%Y-%m-%d_%H:%M:%S')
 GIT_BRANCH=$(shell git rev-parse --abbrev-ref HEAD 2>/dev/null)
 GIT_COMMIT=$(shell git rev-parse --short HEAD 2>/dev/null)
 
+LDFLAGS=-ldflags "\
+	-X github.com/sustainable-computing-io/kepler/internal/version.version=$(VERSION) \
```

Comment:
`version` and `gitCommit` have same value. Version can also have `v.X.Y.Z` information

Reply:
```
VERSION=$(shell git describe --tags --always --dirty || echo "dev")
```
whereas
```
GIT_COMMIT=$(shell git rev-parse --short HEAD 2>/dev/null)
```
they could be same but it will change with release tags.